### PR TITLE
Fallback for outputDir setting & clean up

### PR DESCRIPTION
**Pull request by b3kN**
Implemented basic fallback directory for outputDir in the scenario where it is not set in the capabilities nor the reporter configuration.

My apologies, kicked out the initial PR quickly.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -85,7 +85,8 @@ export default class VideoReporter extends WdioReporter {
       return
     }
 
-    this.#outputDir = runner.config.outputDir as string
+    this.#outputDir = this.options.outputDir ?? runner.config.outputDir as string
+    this.#outputDir = this.#outputDir ?? '_results_'
```
Comment (christian-bromann) [inline src/index.ts:89]: Let's define the default output dir as constant somewhere at the top.

**Review by christian-bromann — COMMENTED** [review]
Can you make sure the tests are passing and that we add a unit test ensuring that there is a default value?

File: src/index.ts (hunk 1 above)
Comment (b3kN) [reply src/index.ts:89]: No problem

**Comment by b3kN** [comment]
> Can you make sure the tests are passing and that we add a unit test ensuring that there is a default value? Also can you make sure to rebase on top of current `main`? Thanks!

Unfortunate - The tests passed locally, but I can adjust the value when setting the default check test.

**Review by christian-bromann — APPROVED** [review]
Awesome, thanks!

**Pull request merged**